### DISCUSSION TITLE
Controlling 2 Apple TVs by using the custom launcher button to change views?

### DISCUSSION THREAD

**Discussion by serbs102** [PRProd/HA-Firemote#483] [Q&A]
Hi - I want to be able to control two ATVs that are controlling two TVs next to each other. I created two remotes with the anticipation that I will use the custom app launcher to change the view cards. 

I'm trying to use a script to change the views, but am not having any luck getting the script to actually change the view/remote. 

Below is the script for the custom launcher
`custom_launchers:
  - friendly_name: Living Room
    label: Living
    color: black
    background: white
  - friend_name: Master
    label: Master
    color: black
    background: white
    script: MATV
app_launch_5: customlauncher Living Room
app_launch_6: customlauncher `Master`

Here's the script I wrote:
`-navigate_to_view:
  alias: Navigate to MATV View
  sequence:
    - service: browser_mod.navigate
      data:
        navigation_path: /lovelace-MATV`

Any idea how to make this work? Or any different idea to control 2 ATVs from one card?

**Comment by PRProd** [comment]
Greetings @serbs102!

The method you are trying will likely not work the way you are hoping.  I would suggest another approach.  If you are against the idea of having 2 Firemotes on your dashboard at once, take a look at [this idea](https://github.com/PRProd/HA-Firemote/discussions/129#discussioncomment-6265750) in the [Let's See Your Dashboards!](https://github.com/PRProd/HA-Firemote/discussions/129) discussion.
